### PR TITLE
feat(argo-cd): Add "global.env" for all deployed containers

### PR DESCRIPTION
**Pull request by joelee2012**
Checklist:

* [x] I have bumped the chart version according to [versioning](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#versioning)
* [x] I have updated the documentation according to [documentation](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#documentation)
* [x] I have updated the chart changelog with all the changes that come with this pull request according to [changelog](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#changelog).
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/blob/master/community/CONTRIBUTING.md).
* [x] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/stable/developer-guide/ci/)).

### CODE REVIEW

**Comment by joelee2012** [comment]
this is for #1987

File: charts/argo-cd/README.md [hunk 1]
```diff
@@ -105,6 +105,9 @@ For full list of changes please check ArtifactHub [changelog].
 
 Highlighted versions provide information about additional steps that should be performed by user when upgrading to newer version.
 
+### 5.36.0
+This version adds `global.env` for all deployed deployments
```
Comment (yu-croco) [inline charts/argo-cd/README.md:109]: This PR does not require specific action (e.g. breaking change) to users, so I think we don't need this note. 🙋

**Comment by yu-croco** [comment]
Please handle this.
> I have updated the chart [changelog](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#changelog) with all the changes that come with this pull request according to changelog.

File: charts/argo-cd/README.md (hunk 1 above)
Comment (joelee2012) [reply charts/argo-cd/README.md]: deleted

**Comment by joelee2012** [comment]
@yu-croco please check again, thanks

**Comment by tico24** [comment]
It appears the changelog still hasn't been updated? https://github.com/argoproj/argo-helm/blob/c81c6599960fd848979eb14a54b35cff0fe713fe/charts/argo-cd/Chart.yaml#L28

**Comment by joelee2012** [comment]
@tico24 You are right, i have updated changelog , can you check now  ?

**Review by tico24 — APPROVED** [review]

**Comment by yu-croco** [comment]
yes I was waiting for https://github.com/argoproj/argo-helm/pull/2096#issuecomment-1579811220 .

**Review by yu-croco — APPROVED** [review]

**Pull request merged**